sympy/physics/vector/point.py

```diff
@@ -571,18 +571,17 @@ def vel(self, frame):
                      'points. This is likely not desired and may cause errors '
                      'in your calculations.')
             if len(candidate_neighbor) > 1:
-                warn('Velocity automatically calculated based on point ' +
-                     candidate_neighbor[0].name +
-                     ' but it is also possible from points(s):' +
-                     str(candidate_neighbor[1:]) +
-                     '. Velocities from these points are not necessarily the '
-                     'same. This may cause errors in your calculations.')
+                warn(f"Velocity of {self.name} automatically calculated based "
+                     f"on point {candidate_neighbor[0].name} but it is also "
+                     f"possible from points(s): {str(candidate_neighbor[1:])}. "
+                     f"Velocities from these points are not necessarily the "
+                     f"same. This may cause errors in your calculations.")
```

Comment:
```suggestion
                warn(filldedent(f"""
                Velocity of {self.name} automatically calculated based
                on point {candidate_neighbor[0].name} but it is also
                possible from points(s): {str(candidate_neighbor[1:])}.
                Velocities from these points are not necessarily the
                same. This may cause errors in your calculations."""))
```

Reply:
`filldedent` can be imported from `sympy.utilities.misc`